src/components/ConfirmationGraphic/index.tsx

```diff
@@ -55,7 +55,7 @@ const ConfirmationContainer = () => {
             <div className="content">{name}</div>
           </div>
           <div className="grid-item">
```

Comment:
We should use the grid-item defined by DS rather then the custom one, and remove the associate css for it.